packages/vant-cli/src/config/webpack.base.ts

```diff
@@ -84,7 +84,7 @@ export const baseConfig: WebpackConfig = {
       },
       {
         test: /\.(js|ts|jsx|tsx)$/,
-        exclude: /node_modules\/(?!(@vant\/cli))/,
+        exclude: /node_modules(\/.pnpm){?}\/(?!(@vant\+cli))/,
```

Comment:
![image](https://user-images.githubusercontent.com/7237365/165305060-fe1932b0-d8c3-44e2-ab3d-2ee902fbc65d.png)

`.pnpm` should be optional:

`/node_modules(\/.pnpm)?\/(?!(@vant\+cli))/`

![image](https://user-images.githubusercontent.com/7237365/165305171-de352555-2112-41f0-9c0d-f2af8bb48bc5.png)

Reply:
hi~ [chenjiahan](https://github.com/chenjiahan), 

after inspection,
`/node_modules(\/\.pnpm)?\/(?!(@vant\/cli))/` cannot be used normally under pnpm,
so use `/^node_modules(\/\.pnpm)?\/(?!(@vant\/cli))/`can work normally.

thank you for taking the time to check.

<img width="967" alt="image" src="https://user-images.githubusercontent.com/18086072/165694136-b9e2486b-512d-481b-b7bd-3ea2d3111934.png">